Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/App.tsx

```diff
@@ -2,9 +2,68 @@ import BatteryInput from "~/components/batteryInput";
 import Header from "~/components/header";
 import SpeedInput from "~/components/speedInput";
 import WeatherInput from "~/components/weatherInput";
+import React, { useState } from "react";
 
 const App = () => {
+  const[speed, setSpeed] = useState<number | string>("");
+  const[battery, setBattery] = useState<number | string>("");
+  const[weather, setWeather] = useState(0);
+  const[range, setRange] = useState(0);
+
+   // Error messages
+  const [speedError, setSpeedError] = useState<string>("")
+  const [batteryError, setBatteryError] = useState<string>("")
```

Comment:
Do we need to create a state variable to show the two error state? Can we just infer it from the three states above without creating new state for this?